production/loki-mixin/dashboards/loki-writes-resources.libsonnet

```diff
@@ -47,18 +47,12 @@ local utils = import 'mixin-utils/utils.libsonnet';
           .addPanel(
             $.containerCPUUsagePanel('CPU', 'ingester'),
           )
-        )
-        .addRow(
```

Comment:
By having all the panels in the same row, will Grafana adjust the dashboards automatically per row, or will it try to place all dashboards in a single row?